src/puppet-wechat.ts

```diff
@@ -1232,7 +1232,7 @@ export class PuppetWeChat extends PUPPET.Puppet {
     toUserName : string,
   ): Promise<WebMessageMediaPayload> {
     const filename = file.name
-    const ext      = path.extname(filename) //  message.ext()
+    const ext      = path.extname(filename).slice(1).toLowerCase() //  message.ext()
```

Comment:
Please keep the `ext` as the same as the `path.extname()` returned(".xxx") because we can keep the ext value to be consistent to include the "." in it.

Reply:
The attachment type problem is caused by '.', so I deleted it. And the `path.extname` is case sensitive.